mods/sethome/init.lua

```diff
@@ -41,11 +49,18 @@ sethome.set = function(name, pos)
 		end
 		output:write(table.concat(data))
 		io.close(output)
-		return true
 	end
-	return true -- if the file doesn't exist - don't return an error.
 end
 
+minetest.register_on_player_receive_fields(function(player, formname, fields)
+	if formname == "sethome:sethomedialog" then
+		if fields.yes then
+			sethome.set(player:get_player_name(), player:get_pos())
+			minetest.chat_send_player(player:get_player_name(), "Home set!")
```

Comment:
```suggestion
			minetest.chat_send_player(player:get_player_name(), S("Home set!"))
```

Reply:
Done!

Reply:
You are using `player:get_player_name()` in the line above, too. You can get the value once only instead:
```lua
local name = player:get_player_name()
-- use name again
minetest.chat_send_player(name, S("Home set!"))
```

Reply:
OK, I did that.